docs/source/tutorial/trackers/using_tensorboard.rst

```diff
@@ -0,0 +1,73 @@
+Using Tensorboard
+=========================
+
+`Tensorboard <https://www.tensorflow.org/tensorboard/>`_ (TB) is a service for tracking experimental results during training.
+It is part of the larger Tensorflow project but can be used independently.
+
+Minimal Pipeline Example
+---------------------------------
+A CSV log file can be generated with the following:
+
+.. code-block:: python
+
+    from pykeen.pipeline import pipeline
+
+    pipeline_result = pipeline(
+        model='RotatE',
+        dataset='Kinships',
+        result_tracker='tensorboard',
+    )
+
+It is placed in a subdirectory of :mod:`pystow` default data directory with PyKEEN called ``tensorboard``,
```

Comment:
Is this a setting baked into the code, or is this dependent on how you ran tensorboard from the command line?

Reply:
Is it the case that the python code interacts directly with the filesystem and then the frontend also interacts with the filesystem, and not that it interacts with tensorboard through an API?

Reply:
So the python code interacts directly with the filesystem and writes the logs out as TFEvent files. These log files are then read by the `tensorboard` process to be displayed in the browser.

To be clear, the tensorboard frontend does not need to be running at the same time as the training. So the `--logdir` argument is simply telling TB where to look for the TFEvent files to display.  These logs can be copied from machine to machine as well - this is often how I use it as I train on the cluster, but want to run TB from my own machine.

Reply:
Really interesting! This is different from how the other trackers for Weights and Biases and MLflow work, so definitely worth including in the tutorial